server/src/main/java/org/spine3/server/projection/ProjectionStorage.java

```diff
@@ -57,6 +58,12 @@ protected void writeRecord(I id, EntityStorageRecord record) {
         storage.write(id, record);
     }
 
+    @Override
+    protected void writeRecords(Map<I, EntityStorageRecord> records) {
+        final RecordStorage<I> storage = getRecordStorage();
```

Comment:
In some places we have `recordStorage()`, here we have the `get` prefix. I think it would be better to have `recordStorage()` everywhere.